### PR TITLE
{tools}[foss/2023a] Optuna v3.5.0

### PR DESCRIPTION
**Pull request by ThomasHoffmann77**
(created using `eb --new-pr`)

### CODE REVIEW

**Review by Micket — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/r/Redis/Redis-7.2.3-GCC-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+homepage = 'https://redis.io'
+description = """Redis is an open source (BSD licensed), in-memory data structure store, used as
+a database, cache, and message broker. Redis provides data structures such as
+strings, hashes, lists, sets, sorted sets with range queries, bitmaps,
+hyperloglogs, geospatial indexes, and streams. Redis has built-in replication,
+Lua scripting, LRU eviction, transactions, and different levels of on-disk
+persistence, and provides high availability via Redis Sentinel and automatic
+partitioning with Redis Cluster."""
+
+toolchain = {'name': 'GCC', 'version': '12.3.0'}
```
[the remark this answers is not in the document]
Comment (ThomasHoffmann77) [reply easybuild/easyconfigs/r/Redis/Redis-7.2.3-GCC-12.3.0.eb:15]: @Micket done

File: easybuild/easyconfigs/r/redis-py/redis-py-5.0.1-foss-2023a.eb [hunk 2]
```diff
@@ -0,0 +1,32 @@
+easyblock = 'PythonBundle'
+
+name = 'redis-py'
+version = '5.0.1'
+
+homepage = 'https://github.com/redis/redis-py'
+description = "The Python interface to the Redis key-value store."
+
+toolchain = {'name': 'foss', 'version': '2023a'}
```
[the remark this answers is not in the document]
Comment (ThomasHoffmann77) [reply easybuild/easyconfigs/r/redis-py/redis-py-5.0.1-foss-2023a.eb]: done

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
@boegelbot please test @ jsc-zen3

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 6 out of 6 (4 easyconfigs in total)
vera40-1 - Linux Rocky Linux 8.9, x86_64, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, 1 x NVIDIA Tesla T4, 550.54.14, Python 3.6.8
See https://gist.github.com/Micket/166d274983fd3499ae0104e5d145d929 for a full test report.

**Comment by Micket** [comment]
Just loading Optuna and it works with CuPy directly, so i'm not sure we need to add this as a strict dep to CuPy, it can be opt-in at no loss for those who actually want to optimize the code evaluation i guess.

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
Going in, thanks @ThomasHoffmann77!

**Pull request merged**